### PR TITLE
DOC: Suggest using 72 characters or less in commit hooks

### PR DESCRIPTION
**Pull request by thewtex**
This ensures the entire summary fits into GitHub's renderings.

Also reference the 78 character max enforced by Git client and pull
request hooks.

Suggest the summary, space, detailed description formatting.

Following [discussion on Discourse](https://discourse.itk.org/t/new-convention-for-commit-messages-on-github/1427/10).

### CODE REVIEW

File: Utilities/Hooks/prepare-commit-msg [hunk 1]
```diff
@@ -38,6 +38,13 @@ instructions='#\
 #   PERF:   - performance improvement\
 #   STYLE:  - no logic impact (indentation, comments)\
 #   WIP:    - Work In Progress not ready for merge\
+#
+# The first line of the commit message should be 72 characters or less; the
+# maximum allowed is 78 characters.
+#
+# Follow the first line commit summary with a space, then a detailed
```
Comment (dzenanz) [inline Utilities/Hooks/prepare-commit-msg]: space -> empty line

File: Utilities/Hooks/prepare-commit-msg [hunk 2]
```diff
@@ -38,6 +38,13 @@ instructions='#\
...
+# description in one or more paragraphs.
```
Comment (bradking) [inline Utilities/Hooks/prepare-commit-msg]: It's not a "space", but an "empty line" or "blank line".

File: Utilities/Hooks/prepare-commit-msg [hunk 3]
```diff
@@ -38,6 +38,13 @@ instructions='#\
 #   PERF:   - performance improvement\
 #   STYLE:  - no logic impact (indentation, comments)\
 #   WIP:    - Work In Progress not ready for merge\
+#
+# The first line of the commit message should be 72 characters or less; the
```
Comment (N-Dekker) [inline Utilities/Hooks/prepare-commit-msg]: Maybe `should preferably be 72 characters...` (The word "preferably" makes it sound less strict, more friendly, to me).

**Comment by thewtex** [comment]
Thanks the reviews. Addressing comments...

**Pull request merged**